source/reference/limitations.txt

```diff
@@ -64,6 +64,38 @@ General Limitations
 - You cannot sync a collection that has a unique index and a non-unique
   index defined on the same field(s).
 
+.. _c2c-limitations_multiple_sync:
+
+Multiple Source Sync Limitations
+--------------------------------
+
+.. include:: /includes/many-with-one-cluster.rst
+
+A namespace is a ``database_name.collection_name`` combination. You can
+only sync namespaces that don't conflict.
```

Comment:
**Praise**

This is a great additional piece of information. My only suggestion would be to add a `..tip::` admonition to draw extra attention to this concept since its pivot to understand what can and cannot be synced.

Reply:
Will leave as is. The tips are shown in distracting boxes.